tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+---
+layout: tutorial
+title: Write Your First Flutter App
+permalink: /tutorials/get-started/
+---
+
+This is a guide to creating your first Flutter app. If you know
+how to program, you can complete this tutorial. You don’t need
+previous experience with Dart or mobile programming.
+
+* TOC
+{:toc}
+
+### What are Flutter and Dart?
+
+_Flutter_ is a mobile app SDK to help developers and designers
+build modern mobile apps for iOS and Android.
+
+_Dart_ is an application programming language from Google that's
+easy to learn, easy to use, and deployable everywhere.
+Flutter apps are written in Dart.
+
+### What you'll build
+
+You’ll implement a simple mobile app that proposes names for a
+startup company. The user can select and unselect proposed names,
+saving the best ones. The code generates ten names at a time.
+As the user scrolls, new batches of names are generated.
+
+<center><img src="images/StartupNameGenerator-screenshot.png" alt="Screenshot for a Startup Name Generator"></center>
+<center>[PENDING: Temp image - replace with video]</center><br>
+
+The following video shows how the finished app works:
+
+[PENDING: Matt is making the video]
```

Comment:
Video is made; did you get the link? Ping me if not.

Reply:
Got it and done!  :D